crates/subspace-farmer/src/bin/subspace-farmer/commands/farm.rs

```diff
@@ -254,6 +254,10 @@ where
         .iter()
         .enumerate()
         .flat_map(|(disk_farm_index, single_disk_plot)| {
+            let disk_farm_index = disk_farm_index
+                .try_into()
+                .expect("More than 256 plots are not supported, what are you even doing?!");
```

Comment:
Such messages should be changed to emotionally neutral, I sympathize with the question though.